devel/icu/Portfile

```diff
@@ -25,16 +27,19 @@ long_description \
     support for supplementary Unicode characters (needed for GB 18030 repertoire support).
 
 homepage                    http://site.icu-project.org/
-master_sites    http://download.icu-project.org/files/${my_name}/${version}/
+master_sites    https://github.com/unicode-org/icu/archive/
+
+distname        release-[join [split ${version} .] -]
```

Comment:
If a port is going to download from GitHub, it should use the github portgroup; then you don't need to manually set `master_sites` (or `distname`, if you're using an automatically-generated tarball, which you are here). To still get the same distfile you're getting here, you would also set `github.tarball_from archive`.